modules/test/playwright/tests/accessibility-menu-web/accessibilityMenu.spec.ts

```diff
@@ -0,0 +1,122 @@
+	apiHelpersTest,
+	featureFlagPagesTest,
+	instanceSettingsPagesTest,
+	loginTest(),
+	usersAndOrganizationsPagesTest
+);
+
+test('Verify that the default value is displayed when the user has never changed the accessibility setting, regardless of the login status', async ({
+	accessibilityMenuPage,
+	apiHelpers,
+	instanceSettingsPage,
+	page,
+}) => {
+	await instanceSettingsPage.goToInstanceSetting(
+		'Accessibility',
+		'Accessibility Menu'
+	);
+
+	await accessibilityMenuPage.enableAccessibilityMenu();
+
+	const userAccount = await apiHelpers.headlessAdminUser.postUserAccount();
+
+	userData[userAccount.alternateName] = {
+		name: userAccount.givenName,
+		password: 'test',
+		surname: userAccount.familyName,
+	};
+
+	try {
+		await test.step('Verify that the "underlined links" toggle is off when logged out, then turn it on', async () => {
+			await performLogout(page);
+
+			await accessibilityMenuPage.openAccessibilityMenu();
+
+			await expect(
+				accessibilityMenuPage.underlinedLinksToggle
+			).not.toBeChecked();
+
+			await accessibilityMenuPage.toggleUnderlinedLinks(true);
+		});
+
+		await test.step('Login as a new user and assert that the "underlined links" guest preference is copied to the user', async () => {
+			await performLogin(page, userAccount.alternateName);
+
+			await accessibilityMenuPage.openAccessibilityMenu();
+
+			await expect(
+				accessibilityMenuPage.underlinedLinksToggle
+			).toBeChecked();
+		});
+
+		await test.step('Verify that the "underlined links" toggle is on when logged out, then turn it off', async () => {
+			await performLogout(page);
+
+			await accessibilityMenuPage.openAccessibilityMenu();
+
+			await accessibilityMenuPage.toggleUnderlinedLinks(false);
+		});
+
+		await test.step('Confirm that the "underlined links" toggle is off when logged in, since user did not change the preference', async () => {
+			await performLogin(page, userAccount.alternateName);
+
+			await accessibilityMenuPage.openAccessibilityMenu();
+
+			await expect(
+				accessibilityMenuPage.underlinedLinksToggle
+			).not.toBeChecked();
+		});
+
+		await test.step('Then, change the "underlined links" preference for that user for the first time', async () => {
+			await accessibilityMenuPage.toggleUnderlinedLinks(true);
+		});
+
+		await test.step('Ensure that the "underlined links" toggle is off after logging out', async () => {
+			await performLogout(page);
+
+			await accessibilityMenuPage.openAccessibilityMenu();
+
+			await expect(
+				accessibilityMenuPage.underlinedLinksToggle
+			).not.toBeChecked();
+		});
+
+		await test.step('Confirm that the "underlined links" toggle is on after logging in', async () => {
+			await performLogin(page, userAccount.alternateName);
+
+			await accessibilityMenuPage.openAccessibilityMenu();
+
+			await expect(
+				accessibilityMenuPage.underlinedLinksToggle
+			).toBeChecked();
+
+			await expect(
+				accessibilityMenuPage.underlinedLinksToggle
+			).toBeChecked();
```

Comment:
Looks like an accidental duplicate of this assertion. Please remove one.

Reply:
Ops, thanks